spec/requests/universal_links_spec.rb

```diff
@@ -14,7 +14,7 @@
 
         both_app_ids = [forem_app_id, dev_app_id]
         expect(response).to have_http_status(:ok)
-        expect(json_response.dig("applinks", "apps")).to match_array(both_app_ids)
+        expect(json_response.dig("applinks", "apps")).to match_array([])
```

Comment:
`match_array` is an order-independent array matcher. For an empty array I'd probably use a different one:

```suggestion
        expect(json_response.dig("applinks", "apps")).to eq([])
```

Reply:
You can also use `.to be_empty` :D

Reply:
Makes sense, thank you for the suggestions! I went with `be_empty` for these 🙂